guides/common/modules/ref_migrating-from-foreman_webhooks.adoc

```diff
@@ -0,0 +1,12 @@
+[id="migrating-webhooks_{context}"]
+= Migrating from Foreman Webhooks
```

Comment:
This title makes it look like it is a procedure. 
Gerunds are used for procedures. 
However I cannot find the procedure in the text. 
Please develop a title to best suit the content and respect the mod doc and minimalist conventions.